src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -71,6 +77,30 @@
     private static final List<Author> AUTHOR_CONFIG_SPECIAL_CHARACTER_AUTHORS = Arrays.asList(
             FIRST_SPECIAL_CHARACTER_AUTHOR, SECOND_SPECIAL_CHARACTER_AUTHOR, THIRD_SPECIAL_CHARACTER_AUTHOR);
 
+    private static final Author FIRST_COMMAS_AND_DOUBLEQUOTES_AUTHOR = new Author("ProcessedCooked");
+    private static final Author SECOND_COMMAS_AND_DOUBLEQUOTES_AUTHOR = new Author("codeeong");
+    private static final Author THIRD_COMMAS_AND_DOUBLEQUOTES_AUTHOR = new Author("jordancjq");
+    private static final String FIRST_COMMAS_AND_DOUBLEQUOTES_DISPLAY_NAME = "Tay Fan Gao, Douya";
+    private static final String SECOND_COMMAS_AND_DOUBLEQUOTES_DISPLAY_NAME = "\"\"Tora, S/O,\" Doyua, T.\"";
```

Comment:
I am a little confuse on how it came to the conclusion that the result, at the front for example, only few of all double quotes were captured. Whats the rationale for it?

Reply:
In MS Excel, every double quote you entered will be "translated" to **two** double quotes in raw format.
![image](https://user-images.githubusercontent.com/35756209/60391289-96ed4600-9b1d-11e9-903d-7bd3996ce583.png)

So `""Tora, S/O," Douya, T."` in excel means `"""""Tora, S/O,"" Douya, T."""` in RAW form.

- Because certain characters are added, such as `,` and `"`, the entire string will be enclosed with a double-quote.
- For each double-quote entered in Excel, it will be **two** double quotes in RAW form.